test/unit/app/test-process-tasks.js

```diff
@@ -180,27 +113,6 @@ describe('process-tasks', function () {
     })
   })
 
-  it('should not update the workload report for UPDATE CALCULATE-WORKLOAD-POINTS tasks', function () {
-    getTaskInProgressCount.resolves([{ theCount: 0 }])
-    getPendingTasksAndMarkInProgress.resolves([
-      { id: 2, workloadReportId: 2, type: 'CALCULATE-WORKLOAD-POINTS', additionalData: { operationType: operationTypes.UPDATE } }
-    ])
-    getWorkerForTask.returns({
-      execute: async function () {
-        return 'Done!'
-      }
-    })
-
-    completeTaskWithStatus.resolves({})
-
-    return processTasks().then(function () {
-      expect(completeTaskWithStatus.called).to.be.true
```

Comment:
Is this single check worth retaining in a test? I think we have deleted all of these checks in this PR

Reply:
there are still two in an above test, https://github.com/ministryofjustice/wmt-worker/blob/main/test/unit/app/test-process-tasks.js#L68...L69